server/src/main/java/org/elasticsearch/TransportVersions.java

```diff
@@ -165,6 +165,7 @@ static TransportVersion def(int id) {
     public static final TransportVersion ADD_INCLUDE_FAILURE_INDICES_OPTION = def(8_831_00_0);
     public static final TransportVersion ESQL_RESPONSE_PARTIAL = def(8_832_00_0);
     public static final TransportVersion RANK_DOC_OPTIONAL_METADATA_FOR_EXPLAIN = def(8_833_00_0);
+    public static final TransportVersion INFERENCE_REQUEST_ADAPTIVE_RATE_LIMITING_ADDED = def(8_834_00_0);
```

Comment:
```suggestion
    public static final TransportVersion INFERENCE_REQUEST_ADAPTIVE_RATE_LIMITING = def(8_834_00_0);
```
Just looking at the other names they just seem to declare the feature.